Modules/Brand/Http/Controllers/BrandController.php

```diff
@@ -5,20 +5,13 @@
 use Illuminate\Http\Request;
 use Illuminate\Routing\Controller;
 use Illuminate\Http\JsonResponse;
-use Illuminate\Support\Facades\Validator;
-
 use Illuminate\Support\Str;
-use Modules\Brand\Entities\Catalog;
-use Modules\User\Entities\User;
-use Modules\Brand\Entities\Brand;
-use Modules\Retailer\Entities\Retailer;
-use Modules\Cart\Entities\Cart;
 use Modules\Brand\Http\Requests\StoreBrandRequest;
 use Modules\Brand\Http\Requests\UpdateBrandRequest;
+use Modules\Brand\Http\Requests\UpdateBrandAccountRequest;
+use Modules\Brand\Http\Requests\UpdateBrandShopRequest;
 use Modules\Brand\Http\Services\BrandService;
-use File;
-use DB;
-use Exception;
+
```

Comment:
Add use Modules\Brand\Entities\Brand;
we are using Brand in liveShop and other d=functions, but not included it.

Reply:
done